app/brave_generated_resources.grd

```diff
@@ -922,7 +922,9 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
       <message name="IDS_IMPORTED_FROM_BOOKMARK_FOLDER" desc="Name for bookmark panel folder imported from another browser">
         Imported from <ph name="BROWSER_NAME">$1<ex>Chrome</ex></ph>.
       </message>
-
+      <message name="IDS_COPY_CLEAN_LINK" desc="The name of the context menu item to copy clean link">
+        Copy clean link address
```

Comment:
We can probably just use "Copy clean link" (no `address`) since the original option is called "Copy link".

Reply:
After discussing with @rmcfadden3 , let's change this to: "Copy clean link (only main URL)"